inference-engine/src/readers/onnx_reader/ie_onnx_reader.cpp

```diff
@@ -65,7 +65,7 @@ CNNNetwork ONNXReader::read(std::istream& model, const std::vector<IExtensionPtr
     return CNNNetwork(ngraph::onnx_import::import_onnx_model(model, readPathFromStream(model)), exts);
 }
 
-INFERENCE_PLUGIN_API(StatusCode) InferenceEngine::CreateReader(IReader*& reader, ResponseDesc *resp) noexcept {
+INFERENCE_PLUGIN_API(StatusCode) InferenceEngineONNXReader::CreateReader(IReader*& reader, ResponseDesc *resp) noexcept {
```

Comment:
@ilyachur looks strange that create function for reader plugins is located inside namespace. 

For inference plugin we don't have it. Can we remove namespace for exported `CreateReader` functions?

Reply:
That's a good question, since it's exported as "C", it does reserve the global name "CreateReader" anyways...

Reply:
Ok, but not to confuse other I suggesting removing namespace :)

Reply:
Do I remove the definition of CreateReader inside the namespace in ie_reader.hpp, it was here before this PR.

If I don't remove it, when i call it internally i have to use explicit global namespace otherwise there is ambiguity at compile time.